barretenberg/cpp/src/barretenberg/aztec_ivc/aztec_ivc.test.cpp

```diff
@@ -185,6 +185,10 @@ TEST_F(AztecIVCTests, BadProofFailure)
         // Construct and accumulate a set of mocked private function execution circuits
         size_t NUM_CIRCUITS = 4;
         for (size_t idx = 0; idx < NUM_CIRCUITS; ++idx) {
+            if (idx == 3) {
```

Comment:
can you add a comment to explain why this is a special case please

Reply:
I'm also a bit confused how this is related to the tampered fold proof

Reply:
the 4th circuit (idx == 3) is the kernel circuit that calls the recursive folding verifier with the folding proofs of the 2nd and 3rd instances. Since we're tampering with the first proof, this circuit now throws an error.

Reply:
thanks for explaining! would you be able to add a comment about this and then I think it's a pretty good failing test. Will approve